signac/contrib/project.py

```diff
@@ -26,7 +26,7 @@
 from ..common.config import Config, get_config, load_config
 from ..core import json
```

Comment:
We'll want to carefully review any remaining references to `core/json.py`.

Reply:
Yes. Half of them are in deprecated modules, the other half (as far as I can tell) don't actually need the one bit of extra functionality provided by `signac/core/json.py` (making it possible to dump `_SyncedDict` objects). We should be able to deprecate `json.py`. I would like to do that in a follow-up PR that also deprecates the old `_SyncedDict` family of classes.

Reply:
That's fine, we can resolve this if you don't think there are any action items yet.